### DISCUSSION TITLE
Capacity on console is inconsistent

### DISCUSSION THREAD

**Discussion by midorinet** [minio/minio#16258] [Q&A] [answered]
My console showing different value everytime I refresh the console, specially on the Capacity ones

1st : 71% free, 0 bytes used
2nd : 213% free, 0 bytes used
3rd : 71% free, 864 GB used

This happens randomly

**Comment by harshavardhana** [comment, marked as answer]
Please report this to https://github.com/minio/console/issues

**Reply by klauspost** [reply]
It already is https://github.com/minio/console/issues/2506
